CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Change Log
 ==========
 
+Version 0.18 *(2015-07-14)*
+------------------
+ * Destroying a scope recursively destroys its children first, like it used to.
+   (0.17 API quake incorrectly reversed this.)
+
+ * Now crashes (fail fast!) when doing service lookup in a dead scope. 
+
+ * Manually fall back to app context when service not found. Works around
+   window in activity context life cycle where they're not quite attached yet.
```

Comment:
"Works around window in activity context" reads like there are some words missing.  Also "they" doesn't really agree with its antecedent.